internal/services/instance_types_service_test.go

```diff
@@ -64,5 +64,24 @@ func TestListInstanceTypesHandler(t *testing.T) {
 
 		assert.Contains(t, rr.Body.String(), "missing parameter")
 	})
+}
+
+func TestListAzureBuiltinInstanceTypesHandler(t *testing.T) {
+	ctx := context.Background()
+
+	req, err := http.NewRequestWithContext(ctx, "GET", "/api/provisioning/v1/instance_types/azure", nil)
```

Comment:
We should probably pull the prefix out into a constant for all the tests, good for now.